packages/arb-token-bridge-ui/src/components/TransferPanel/WithdrawalConfirmationDialog.tsx

```diff
@@ -24,14 +24,20 @@ const SECONDS_IN_DAY = 86400
 const SECONDS_IN_HOUR = 3600
 
 function getCalendarUrl(
-  confirmationDays: number,
+  confirmationHours: number,
   amount: string,
   token: string,
   networkName: string
 ) {
   const title = `${amount} ${token} Withdrawal from ${networkName}`
-  const withdrawalDate = dayjs().add(confirmationDays, 'day').format('YYYYMMDD')
-  return `https://calendar.google.com/calendar/render?action=TEMPLATE&text=${title}&dates=${withdrawalDate}%2F${withdrawalDate}`
+
+  const withdrawalDate = dayjs().add(confirmationHours, 'hour')
+  // Google event date format: YYYYMMDDTHHMMSS/YYYYMMDDTHHMMSS
+  const parsedWithdrawalDate = withdrawalDate.format(
+    'YYYYMMDD[T]HH[0000%2F]YYYYMMDD[T]HH[0000]'
+  )
```

Comment:
Seems like we're not accounting for minutes and always setting them to 0? I wanted to initiate a withdrawal at 11:30, and the Google Calendar event was being scheduled for 15:00, which is 3.5 hours instead of 4.

Reply:
@fionnachan yep we could do 5 minutes instead of 1 hour for more precision. Thoughts @spsjvc ?